spec/models/course/lesson_plan/lesson_plan_item_spec.rb

```diff
@@ -99,5 +99,63 @@
         end
       end
     end
+
+    describe 'personal times' do
+      let(:student1) { create(:course_student, course: course) }
+      let(:student2) { create(:course_student, course: course) }
+      let(:personal_time) do
+        personal_time = lesson_plan_item.find_or_create_personal_time_for(student1)
+        personal_time.save!
+        personal_time
+      end
+
+      it 'creates a personal time for course_user' do
```

Comment:
This also initialises the start/end/bonus times to be equal to the reference time right? Can you check for that too?